### PR TITLE
podman remote client -- add rmi

### PR DESCRIPTION
**Pull request by baude**
allow the podman remote client to delete images

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: cmd/podman/rmi.go [hunk 1]
```diff
@@ -58,11 +57,11 @@ func rmiCmd(c *cli.Context) error {
 		return err
 	}
```
Comment (jwhonce) [inline cmd/podman/rmi.go]: Sort order

File: cmd/podman/rmi.go [hunk 2]
```diff
@@ -58,11 +57,11 @@ func rmiCmd(c *cli.Context) error {
...
 	removeAll := c.Bool("all")
-	runtime, err := libpodruntime.GetRuntime(c)
+	localRuntime, err := adapter.GetRuntime(c)
```
Comment (jwhonce) [inline cmd/podman/rmi.go]: Isn't runtime more descriptive here than localRuntime? Or am I wrong that adapter could return remoteRuntime?

**Comment by jwhonce** [comment]
@baude No tests?

File: cmd/podman/rmi.go (hunk 2 above)
Comment (baude) [reply cmd/podman/rmi.go]: yes, but no solid, agreeable name has been determined.  lets work this out as we go.

**Comment by baude** [comment]
> @baude No tests?

I've pushed a sep commit from another branch I had so they are here now ... I'm not sure if we should combine them or leap frog due to size.  But this was reasonable in size.

File: cmd/podman/rmi.go (hunk 2 above)
Comment (rhatdan) [inline cmd/podman/rmi.go]: I would just leave this as runtime, err :=

Comment (rhatdan) [reply cmd/podman/rmi.go]: You are calling GetRuntime() so call it runtime.  Adding something like activeruntime does not make it any more descriptive to me.  I just think of both as a runtime, and adding more characters to it does not help.  I think you guys are overthinking this.  The runtime is either a libpod runtime or a varlink runtime.

**Comment by rhatdan** [comment]
Fix up all the import mess and change back to runtime, and we can merge.

File: cmd/podman/rmi.go (hunk 2 above)
Comment (baude) [reply cmd/podman/rmi.go]: ok, done, will catch others in next pr

**Pull request merged**